client/state/sites/test/reducer.js

```diff
@@ -412,7 +412,7 @@ describe( 'reducer', () => {
 				}
 			} );
 
-			expect( state ).to.equal( original );
+			expect( state ).to.eql( original );
```

Comment:
Since we're now returning memo, which is a copy of state, it was necessary to change these to `eql`.